### ISSUE TITLE
[BUG] Kindling Love

### ISSUE THREAD

**Issue by FelixArvg** [DaFuqs/Spectrum#408]
**Describe the bug**
Crash game

**To Reproduce**
Just break a fully grown flower from the world of DD in Overworld

**Expected behavior**
Explosion from a flower

**Minecraft version**
Fabric 1.19.2

**Mod version**
1.7.6 and 1.7.11

**Screenshots**
![image](https://github.com/DaFuqs/Spectrum/assets/170378954/caa6e186-6492-4701-9b64-ca6810fe8f7a)

**Crash Report**
[crash-2024-05-31_11.31.12-server.txt](https://github.com/user-attachments/files/15541037/crash-2024-05-31_11.31.12-server.txt)

**Comment by DaFuqs** [comment]
On first glance, this looks like a version mismatch. Are client and server running the same version of Spectrum?

**Comment by FelixArvg** [comment]
> On first glance, this looks like a version mismatch. Are client and server running the same version of Spectrum?

If there is the same version of the mod as on the server, then a disconnect is provided when connecting to the server

**Comment by DaFuqs** [comment]
Interestingly, the error states something else as the issue from the screenshot entirely. 

```
java.lang.NoClassDefFoundError: dev/ftb/mods/ftbchunks/data/FTBChunksAPI
	at dev.ftb.mods.ftbxmodcompat.fabric.ftbchunks.commonprot.FTBChunksProtectionProvider.canBreakBlock(FTBChunksProtectionProvider.java:85)
```

that rings a bell. See: https://github.com/DaFuqs/Spectrum/issues/388#issuecomment-2097803370

TL;DR: Update FTB chunks.
